src/index/MetaDataHandler.h

```diff
@@ -0,0 +1,199 @@
+// Copyright 2018, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Johannes Kalmbach (johannes.kalmbach@gmail.com)
+//
+#pragma once
+
+#include <stxxl/vector>
+#include "../util/HashMap.h"
+#include "../global/Id.h"
+#include "../util/Exception.h"
+#include "../util/Log.h"
+
+// _____________________________________________________________
+// hidden in implementation namespace
+namespace ExtVecWrapperImpl {
```

Comment:
It's not really important that `ExtVec` is external, right? So why not just call it `Vec`?